self_hosting_machinery/webgui/selfhost_fastapi_completions.py

```diff
@@ -281,11 +298,11 @@ async def _coding_assistant_caps(self):
             "endpoint_embeddings_style": "openai",
             "size_embeddings": 768,
 
-            "tokenizer_path_template": "https://huggingface.co/$MODEL/resolve/main/tokenizer.json",
+            "tokenizer_path_template": "/get-tokenizer/$MODEL/tokenizer.json",
             "tokenizer_rewrite_path": {
-                model: self._model_assigner.models_db[model]["model_path"]
+                model: file.replace(env.DIR_WEIGHTS + "/", "").replace("/tokenizer.json", "")
                 for model in models_available
-                if model in self._model_assigner.models_db
+                if (file := tokenizer_json_file_for_model(self._model_assigner.models_db.get(model, {})))
             },
```

Comment:
looks like we don't need to rewrite tokenizer path because backend knows it's place for given model
this hack was for loading tokenizer from huggingface where models have different names